plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteJdbcMetadataFactory.java

```diff
@@ -30,15 +31,16 @@ public class IgniteJdbcMetadataFactory
     private final Set<JdbcQueryEventListener> jdbcQueryEventListeners;
 
     @Inject
-    public IgniteJdbcMetadataFactory(JdbcClient jdbcClient, Set<JdbcQueryEventListener> jdbcQueryEventListeners)
+    public IgniteJdbcMetadataFactory(JdbcClient jdbcClient, Set<JdbcQueryEventListener> jdbcQueryEventListeners,
+            SyntheticColumnHandleBuilder builder)
```

Comment:
when we break args to newlines we tend to do it for all the args - each on a newline.

Reply:
```suggestion
    public IgniteJdbcMetadataFactory(
            JdbcClient jdbcClient,
            Set<JdbcQueryEventListener> jdbcQueryEventListeners,
            SyntheticColumnHandleBuilder builder)
```